ramls/time-period.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
```

Comment:
I think raml-cop will complain if there is not a description here. Yup, just looked at the raml-cop comment in the PR and saw this: `ERROR: lint-raml-cop: time-period.json: Missing top-level "description".`